frontend/website/src/Nav.re

```diff
@@ -380,6 +380,11 @@ let make = (~page) => {
         link="https://github.com/CodaProtocol/coda"
         activePage=false
       />,
+      <SimpleButton
```

Comment:
does this look crowded on a 13 in screen? should we remove github

Reply:
I'd guess it collapses to a dropdown before it gets too crowded

Reply:
keeping the link